crates/kitsune_p2p/kitsune_p2p/src/types/agent_store.rs

```diff
@@ -12,21 +12,26 @@ pub type Urls = Vec<Url2>;
 /// Value in the peer database that tracks an Agent's representation as signed by that agent.
 #[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, derive_more::AsRef)]
 pub struct AgentInfoSigned {
```

Comment:
note the changes here that make signed info match what the bootstrap expects

importantly the _serialized_ agent info data is on this struct as the _exactly bytes that were signed_ alongside the pubkey for the signature, to allow for context-free, unambiguous signature checking _before_ any deserialization is attempted (e.g. we don't rely on messagepack conventions to determine bytes as a validator)